server/src/main/java/org/elasticsearch/action/search/SearchResponseSections.java

```diff
@@ -31,6 +52,8 @@ public class SearchResponseSections {
     protected final Boolean terminatedEarly;
     protected final int numReducePhases;
 
+    private final RefCounted refCounted;
```

Comment:
I'm unsure what the point of this is. Is this a placeholder for hits, as indicated by the comment below "`// TODO: noop until hits are ref counted`"?  If so, a name or comment to this effect could be helpful.